<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlamConfig</class>
 <widget class="QDialog" name="SlamConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SLAM机器人设置</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>信息检索</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入要检索的机器人名称</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
    <widget class="QPushButton" name="del">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>选中删除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>列表</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>800</width>
       <height>560</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>机器人登记</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>81</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>机器人IP地址：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_0">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入机器人名称</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>360</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>290</y>
       <width>81</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>雷达控制板IP：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>360</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>81</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>激光雷达IP：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>220</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>机载网络客户端的IP地址</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入机器人日常的存放位置</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>81</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>所属部门：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>190</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>新增</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>81</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>网络客户端IP：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>81</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>摄像头IP地址：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入机器人的日常管理部门</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>81</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>摄像头名称：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>日常存放位置：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>230</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>编辑</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>320</y>
       <width>81</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>平板电脑IP：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入机载摄像头的名称</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>310</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>输入机载平板板电脑的IP地址</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>280</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>激光雷达控制板Navicat的IP地址</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>20</width>
       <height>341</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>机器人名称：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入机载摄像头的IP地址</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>22</width>
       <height>341</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>250</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>激光雷达的IP地址</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>191</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ff0000;&quot;&gt;重要提示：&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ff0000;&quot;&gt;本窗口中有关IP地址的操作&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ff0000;&quot;&gt;错误会造成无法远程控制机&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ff0000;&quot;&gt;器人，请慎重操作！！！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_0</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>lineEdit_8</tabstop>
  <tabstop>lineEdit_9</tabstop>
  <tabstop>lineEdit_10</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_3</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>SlamConfig</receiver>
   <slot>on_cancel_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SlamConfig</receiver>
   <slot>on_save_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>SlamConfig</receiver>
   <slot>on_new_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_save_clicked()</slot>
  <slot>on_new_clicked()</slot>
  <slot>on_cancel_clicked()</slot>
 </slots>
</ui>
